src/main/io/gps.c

```diff
@@ -989,7 +1003,7 @@ static bool UBLOX_parse_gps(void)
         //i2c_dataset.time                = _buffer.posllh.time;
         gpsSol.llh.lon = _buffer.posllh.longitude;
         gpsSol.llh.lat = _buffer.posllh.latitude;
-        gpsSol.llh.alt = _buffer.posllh.altitude_msl / 10 / 100;  //alt in m
+        gpsSol.llh.alt = _buffer.posllh.altitude_msl / 100;  // alt in 0.1m resolution (formerly: 1m resolution)
```

Comment:
This will break altitude in all places where `llh.alt` is used. These will all have to be amended to take the new scaling into account.

Reply:
Yeah, i was afraid of that. It does no make sense, unless differential GPS is used.
Will be reverted.